app/Http/Resources/SimpleSaveResource.php

```diff
@@ -30,6 +30,9 @@ public function toArray($request)
             "owner" => new SimplestUserResource($this->owner),
             "owner_deleting" => $this->owner->trashed(),
             "tool_id" => $this->tool_id,
```

Comment:
could we change this to ToolResource? That would allow us to create a **SimpleSaveResource** off of an **SimplerSaveResource**, since an **SimpleSaveResource** is basically a merge of an **SimplerSaveResource** + some new attributes. The inheritance will then be quite nice, even tho this change will require some code changes in the frontend.

`SimplerSaveResource` -> `SimpleSaveResource` -> `SaveResource`

Reply:
Just saw this was already added on the [feature/openapi-docu](https://github.com/ricom/toolbox-backend/tree/feature/openapi-docu) branch so forget about everything 😄